### PR TITLE
[demo] Use submenus for new BuyersGuide menu

### PR DESCRIPTION
**Pull request by KalobTaulien**
:warning: do not merge :warning:

Example code for wagtail menu/submenu nav item generation

### CODE REVIEW

File: network-api/networkapi/buyersguide/admin.py [hunk 1]
```diff
@@ -70,8 +70,8 @@ class WagtailBuyersGuideCategoryAdmin(ModelAdmin):
     add_to_settings_menu = False
     exclude_from_explorer = False
 
-    list_display = ('featured', 'title', 'description')
-    search_fields = ('title')
+    list_display = ('featured', 'name', 'description')
+    search_fields = ('name',)
```
Comment (KalobTaulien) [inline network-api/networkapi/buyersguide/admin.py:74]: I noticed this was throwing an error when viewing this page. `title` -> `name`

File: network-api/networkapi/buyersguide/admin.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
-from django import forms
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
```
Comment (TheoChevalier) [inline network-api/networkapi/buyersguide/admin.py:2]: Strings from this page, as they are only visible in the CMS, shouldn’t be localized through Pontoon, so you will want to either hardcode them, or add this page to the excluded files from localization https://github.com/mozilla/foundation.mozilla.org/blob/master/tasks.py#L23-L29

**Review by TheoChevalier — CHANGES_REQUESTED** [review]

File: network-api/networkapi/buyersguide/admin.py (hunk 2 above)
Comment (Pomax) [reply network-api/networkapi/buyersguide/admin.py:2]: I'm marking this PR as [demo]: this is not a true PR

**Review by Pomax — COMMENTED** [review]
this should be useful when we redo the buyers guide menu - for now we're taking that menu out just to make sure staff don't get confused about where they should be updating/editing products.

File: network-api/networkapi/buyersguide/admin.py (hunk 2 above)
Comment (TheoChevalier) [reply network-api/networkapi/buyersguide/admin.py:2]: Sorry for jumping the gun!

**Pull request closed without merging**